adaptors_task/student/include/impl.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 #include <string>
+#include <stack>
+
+using namespace std;
```

Comment:
No no no no!
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file